cmd/addimport.go

```diff
@@ -542,7 +552,10 @@ func (p *AddImportParams) createImport() *jwt.Import {
 	if p.service {
 		im.Type = jwt.Service
 		im.Share = p.share
+	} else if p.allowTrace {
+		im.AllowTrace = true
 	}
+
```

Comment:
I don't see also where you set the defaults like you did for the service export.

Reply:
works differently on the other you can edit, so it has to have some default value. On the import it is one shot, you cannot edit.

Reply:
adding an import is more complex than an export, because they can simply provide an activation.

Reply:
ok